ansible/templates/usb/scripts/headnode.sh.j2

```diff
@@ -33,10 +33,13 @@ SVC_CERT_FILE_NAME="dc-erigonesd.pem"
 SVC_CERT_FILE="/opt/custom/etc/${SVC_CERT_FILE_NAME}"
 QUERY_CFGDB="${ERIGONES_HOME}/bin/query_cfgdb"
 TIMEOUT="/opt/local/bin/timeout"
+PWD_TMPSTORE="/opt/custom/etc/esdc_svc_tmp_pwds.json"
 
 # certificate for DC internal services 
 SVC_CERT=
 SVC_KEY=
+SVC_CERT_TMPFILE="/opt/custom/etc/esdc_svc_tmp_cert.pem"
```

Comment:
Can't we store it `/tmp` if it is a temp file?
(i.e. do we want to support resume of installation after a reboot?)

Reply:
We definitely want to support the resume. It happes relatively often that vmadm chokes for some reason on slow servers and I really don't want to re-start the whole installation because of this.

/tmp destination is possible but with permanent storage, you can tell the customer just to reboot the box to resume the installation. Otherwise you need to give specific instructions how to clear the service.

Reply:
👍